src/testdir/test_vim9_class.vim

```diff
@@ -3,6 +3,28 @@
 source check.vim
 import './vim9.vim' as v9
 
+def Test_abstract_method()
```

Comment:
There is already an existing Test_abstract_method() test function that tests these cases.  What new test is added in this function?  Also, this function should not be first function in this file.  Also this should be added to the existing Test_abstract_method() function.